pkg/model/context.go

```diff
@@ -190,3 +192,33 @@ func (m *KopsModelContext) UsePrivateDNS() bool {
 
 	return false
 }
+
+// KubernetesVersion parses the semver version of kubernetes, from the cluster spec
+func (c *KopsModelContext) KubernetesVersion() (semver.Version, error) {
```

Comment:
This is copy-pasta from https://github.com/kubernetes/kops/blob/master/pkg/model/components/context.go#L32.  I'll add a comment saying to dedup